car-utility/README.md

```diff
@@ -0,0 +1,39 @@
+## Overview
```

Comment:
I can go through these instructions tomorrow when the rest of my hardware arrives.